sdk/keyvault/keyvault-admin/README.md

```diff
@@ -0,0 +1,69 @@
+# Azure Key Vault Secret client library for JavaScript
```

Comment:
Am ok doing the readme update in a different PR, but let's remove the current content here and keep only the headers.